packages/mux-player/src/index.jsx

```diff
@@ -0,0 +1,413 @@
+      <MediaControlBar>
+        <MediaTimeRange></MediaTimeRange>
+        <MediaTimeDisplay show-duration remaining></MediaTimeDisplay>
+      </MediaControlBar>
+      <MediaControlBar>
+        <MediaMuteButton></MediaMuteButton>
+        <MediaVolumeRange></MediaVolumeRange>
+        <Spacer />
+        <MediaCaptionsButton></MediaCaptionsButton>
+        <MediaPipButton></MediaPipButton>
+        <MediaFullscreenButton></MediaFullscreenButton>
+        {hasAirPlay && <AirPlayButton onClick={onAirPlaySelected} />}
+      </MediaControlBar>
+    </>
+  );
+};
+
+export const VodChromeLarge = (props) => {
+  const { onAirPlaySelected, hasAirPlay = false } = props;
+  return (
+    <>
+      {/* <div slot="centered-chrome">
+        <MediaSeekBackwardButton></MediaSeekBackwardButton>
+        <MediaPlayButton></MediaPlayButton>
+        <MediaSeekForwardButton></MediaSeekForwardButton>
+      </div> */}
+      {/* <MediaControlBar>
+        <MediaTimeRange></MediaTimeRange>
+        <MediaTimeDisplay show-duration remaining></MediaTimeDisplay>
+      </MediaControlBar> */}
+      <MediaControlBar>
+        <MediaPlayButton></MediaPlayButton>
+        <MediaSeekForwardButton></MediaSeekForwardButton>
+        <MediaSeekBackwardButton></MediaSeekBackwardButton>
+        <MediaMuteButton></MediaMuteButton>
+        <MediaVolumeRange></MediaVolumeRange>
+        <MediaTimeRange></MediaTimeRange>
+        <MediaTimeDisplay
+          show-duration
+          remaining
+          style={{ color: "inherit" }}
+        ></MediaTimeDisplay>
+        <MediaCaptionsButton></MediaCaptionsButton>
+        <MediaPipButton></MediaPipButton>
+        <MediaFullscreenButton></MediaFullscreenButton>
+        {hasAirPlay && <AirPlayButton onClick={onAirPlaySelected} />}
+      </MediaControlBar>
+    </>
+  );
+};
+
+export const LiveChromeSmall = (props) => {
+  const { onAirPlaySelected, hasAirPlay = false } = props;
+  return (
+    <>
+      <div
+        slot="centered-chrome"
+        style={{
+          "--media-background-color": "none",
+          "--media-control-background": "none",
+          "--media-button-icon-width": "100%",
+          width: "100%",
+          display: "flex",
+          flexFlow: "row",
+          alignItems: "center",
+          justifyContent: "center",
+        }}
+      >
+        <MediaPlayButton style={{ padding: 0, width: "20%" }}></MediaPlayButton>
+      </div>
+      <MediaControlBar>
+        <MediaMuteButton></MediaMuteButton>
+        <MediaVolumeRange></MediaVolumeRange>
+        <Spacer />
+        <MediaCaptionsButton></MediaCaptionsButton>
+        <MediaPipButton></MediaPipButton>
+        <MediaFullscreenButton></MediaFullscreenButton>
+        {hasAirPlay && <AirPlayButton onClick={onAirPlaySelected} />}
+      </MediaControlBar>
+    </>
+  );
+};
+
+export const LiveChromeLarge = (props) => {
+  const { onAirPlaySelected, hasAirPlay = false } = props;
+  return (
+    <>
+      <MediaControlBar>
```

Comment:
fine for now but I think these should probably look more like web-component usage (even if they're being rendered w/the light vdom). We can discuss.